main.cpp

```diff
@@ -13,7 +13,17 @@ New/This/Pointers/References conclusion
      on the lines below it, write a struct named 'HeapA' that correctly shows how to own an instance of 'A' 
          on the heap without leaking, without using smart pointers. 
  */
-
+struct A {};
+struct HeapA
+{
+    HeapA() { myA = new A; }
+    ~HeapA()
+    {
+        if ( myA != nullptr )
+            delete myA;
+    } 
```

Comment:
you can `delete` pointers that are `nullptr` without issue.
There is no need for the nullptr check here.
- [ ] revise